packages/pwa-kit-runtime/src/ssr/server/build-remote-server.test.js

```diff
@@ -6,7 +6,6 @@
  */
 import {once, RemoteServerFactory} from './build-remote-server'
 import {X_ENCODED_HEADERS} from './constants'
-import awsServerlessExpress from 'aws-serverless-express'
```

Comment:
Same as above. This change might be valid, but anything outside of `template-retail-react-app` forces us to think hard about whether it's a breaking change